ros_cross_compile/ros_cross_compile.py

```diff
@@ -141,6 +142,15 @@ def parse_args(args: List[str]) -> argparse.Namespace:
         default=[],
         nargs='+',
         help='Skip specified rosdep keys when collecting dependencies for the workspace.')
+    parser.add_argument(
+        '--custom-metric-file',
+        required=False,
+        default='{}.json'.format(datetime.now().strftime('%s')),
+        type=str,
+        help='Provide a filename to store the collected metrics. If no name is provided, '
+             'then the filename will be the current time in UNIX Epoch format. In addition, '
+             'the filename must include the JSON file extension. '
```

Comment:
Do we want to validate this somehow?

Reply:
oops -- I forgot to remove the last part. No validation should be necessary.